pages/1.13/security/oss/user-account-management/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: layout.pug
+navigationTitle:  User Account Management
+title: User Account Management
+menuWeight: 10
+excerpt: Managing DC/OS user accounts
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+DC/OS is designed as a multi-user system.
+
+Open DC/OS can handle three types of user accounts:
+
+* **External user accounts**: External user accounts exist for users who want to use their Google, GitHub, or Microsoft credentials. DC/OS does never receive or store the passwords of external users.
+* **Local user accounts**: Local user accounts exist for users who want to create a user account within DC/OS. Usernames and password hashes are stored in CockroachDB on the master nodes.
+* **Service accounts**: Service accounts exist for long-running services that need to refresh authentication tokens autonomously. Public key information is stored per service in CockroachDB within DC/OS.
```

Comment:
A machine or script interacting with a DC/OS cluster should always use a service account

Same consideration as expressed before about "in CockroachDB within DC/OS.".

Reply:
It's difficult to refer the `local` user account as just being a local `user account`.
That's why I went with three types of accounts:
- External user accounts
- Local user accounts
- Service accounts
In this terminology `user` means `human` :)

Reply:
The fact that service accounts are being stored locally is nice to know but it is not their defining characteristic IMO.